### PR TITLE
Added Delivery Policy to the endpoint entity in the new API version

### PR DESCRIPTION
**Pull request by prakharsharma10**
In this PR, we have 

1.  Upgraded the API version, 2017-10-12.
2. Added a new property  'Delivery Policy' on the endpoint property. Note that the design of this new property has already been approved by the ARM team. 

Please refer to the earlier version 2017-04-02 of the swagger to compare the changes made in this PR. 

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/cdn.json [hunk 1]
```diff
@@ -0,0 +1,3186 @@
...
+      }
+    },
+    "ProfileListResult": {
+      "description": "Result of the request to list profiles. It contains a list of profile objects and a URL link to get the the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/cdn.json]: Consider making enums expandable if more could be added in the future. You can use "modeAsString".

**Review by jianghaolu — APPROVED** [review]

**Comment by Lichard** [comment]
@jianghaolu what else needs to be done before this can be merged?

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ListByProfile.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                        {
+                            "name": "www-bing-com",
+                            "properties": {
+                                "hostName": "www.bing.com",
+                                "httpPort": 80,
+                                "httpsPort": 443
+                            }
+                        }
+                    ],
+                    "customDomains": [],
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ListByProfile.json]: This property is not listed in the "EnpointProperties"

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Get.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                {
+                    "name": "www-bing-com",
+                    "properties": {
+                        "hostName": "www.bing.com",
+                        "httpPort": 80,
+                        "httpsPort": 443
+                    }
+                }
+            ],
+            "customDomains": [],
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Get.json]: This property is not listed in the "EnpointProperties"

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                "httpPort": 80,
+                "httpsPort": 443
+            }
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "201": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json]: Please add responses for 200 and 202

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                {
+                    "name": "www-bing-com",
+                    "properties": {
+                        "hostName": "www.bing.com",
+                        "httpPort": 80,
+                        "httpsPort": 443
+                    }
+                }
+            ],
+            "customDomains": [],
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json]: This property is not listed in "EndpointProperties"

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+            "contentTypesToCompress": [],
+            "isCompressionEnabled": false,
+            "optimizationType": null,
+            "probePath": null,
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json]: Example response property values cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    "201": {
+      "body": {
+        "name": "endpoint1",
+        "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1",
+        "type": "Microsoft.Cdn/profiles/endpoints",
+        "tags": {},
+        "location": "WestCentralUs",
+        "properties": {
+            "hostName": "endpoint1.azureedge.net",
+            "originHostHeader": null,
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json]: Cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+            "provisioningState": "Creating",
+            "resourceState": "Creating",
+            "isHttpAllowed": true,
+            "isHttpsAllowed": true,
+            "queryStringCachingBehavior": "IgnoreQueryString",
+            "originPath": null,
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Create.json]: Cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Update.json [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "endpointUpdateProperties": {
+      "tags": {
+        "additionalProperties": "Tag1"
+      }
+    }
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Update.json]: Please add 202 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Update.json [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+      "body": {
+        "name": "endpoint1",
+        "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1",
+        "type": "Microsoft.Cdn/profiles/endpoints",
+        "tags": {
+            "additionalProperties": "Tag1"
+        },
+        "location": "WestCentralUs",
+        "properties": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Update.json]: Please fix the payload according to the comments in create & get

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Delete.json [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1"
+  },
+  "responses": {
+    "202": {
+      "body": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Delete.json]: Remove body and add 204 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Start.json [hunk 12]
```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1"
+  },
+  "responses": {
+    "202": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Start.json]: Please add 200 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Start.json [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+      "body": {
+        "hostName": "endpoint1.azureedge.net",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Start.json]: This payload doesn't match definition "Endpoint".

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Stop.json [hunk 14]
```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1"
+  },
+  "responses": {
+    "202": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Stop.json]: Please add 200 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Stop.json [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+      "body": {
+        "hostName": "endpoint1.azureedge.net",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_Stop.json]: This payload doesn't match definition "Endpoint".

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_PurgeContent.json [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "contentFilePaths": {
+        "contentPaths": ["/folder1"]
+    }
+  },
+  "responses": {
+    "202": {
+      "body": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_PurgeContent.json]: Remove "body".

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_LoadContent.json [hunk 17]
```diff
@@ -0,0 +1,18 @@
...
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "contentFilePaths": {
+        "contentPaths": ["/folder1"]
+    }
+  },
+  "responses": {
+    "202": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_LoadContent.json]: Please add 200 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_LoadContent.json [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+      "body": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_LoadContent.json]: Remove body.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ValidateCustomDomain.json [hunk 19]
```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainProperties": {
+        "hostname": "www.someDomain.com"
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ValidateCustomDomain.json]: "hostName"

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ValidateCustomDomain.json [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "customDomainValidated": true,
+        "message": null,
+        "reason": null
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ValidateCustomDomain.json]: Response properties cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ListResourceUsage.json [hunk 21]
```diff
@@ -0,0 +1,29 @@
+{
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1"
+  },
+  "responses": {
+    "202": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Endpoints_ListResourceUsage.json]: 202 is not a valid response status code in the spec.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_ListByEndpoint.json [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+          "value": [
+            {
+              "name": "www-someDomain-net",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_ListByEndpoint.json]: "location" missing.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_ListByEndpoint.json [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
+              "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1/origins/www-someDomain-net",
+              "type": "Microsoft.Cdn/profiles/endpoints/origins",
+              "properties": {
+                "provisioningState": "Succeeded",
+                "resourceState": "Active",
+                "hostName": "www.someDomain.net",
+                "httpPort": null,
+                "httpsPort": null
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_ListByEndpoint.json]: Avoid using null integers in examples.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_Get.json [hunk 24]
```diff
@@ -0,0 +1,26 @@
...
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "originName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
+      "body": {
+          "name": "www-someDomain-net",
+          "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1/origins/www-someDomain-net",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_Get.json]: Please address the same comment in Origins_ListByEndpoint.json

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_Update.json [hunk 25]
```diff
@@ -0,0 +1,30 @@
...
+    "originName": "www-someDomain-net",
+    "originUpdateProperties": {
+      "httpPort": 42,
+      "httpsPort": 43
+    }
+  },
+  "responses": {
+    "202": {
+      "body": {
+          "name": "www-someDomain-net",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_Update.json]: "location" missing

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_Update.json [hunk 26]
```diff
@@ -0,0 +1,30 @@
...
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "originName": "www-someDomain-net",
+    "originUpdateProperties": {
+      "httpPort": 42,
+      "httpsPort": 43
+    }
+  },
+  "responses": {
+    "202": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/Origins_Update.json]: 202 is not a valid response status code in the spec.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_ListByEndpoint.json [hunk 27]
```diff
@@ -0,0 +1,30 @@
...
+                "name": "www-someDomain-net",
+                "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1/customdomains/www-someDomain-net",
+                "type": "Microsoft.Cdn/profiles/endpoints/customdomains",
+                "properties": {
+                    "provisioningState": "Succeeded",
+                    "resourceState": "Active",
+                    "hostName": "www.someDomain.net",
+                    "customHttpsProvisioningState": "Disabled",
+                    "customHttpsProvisioningSubstate": "None",
+                    "validationData": null
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_ListByEndpoint.json]: Example response property values cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Get.json [hunk 28]
```diff
@@ -0,0 +1,27 @@
...
+        "name": "www-someDomain-net",
+        "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1/customdomains/www-someDomain-net",
+        "type": "Microsoft.Cdn/profiles/endpoints/customdomains",
+        "properties": {
+            "provisioningState": "Succeeded",
+            "resourceState": "Active",
+            "hostName": "www.someDomain.net",
+            "customHttpsProvisioningState": "Disabled",
+            "customHttpsProvisioningSubstate": "None",
+            "validationData": null
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Get.json]: Example response property values cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_EnableCustomHttps.json [hunk 29]
```diff
@@ -0,0 +1,16 @@
...
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
+      "body": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_EnableCustomHttps.json]: Remove empty body

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Create.json [hunk 30]
```diff
@@ -0,0 +1,30 @@
...
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net",
+    "customDomainProperties": {
+      "hostName": "www.someDomain.net"
+    }
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Create.json]: Please add 201 and 202 responses.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Create.json [hunk 31]
```diff
@@ -0,0 +1,30 @@
...
+        "name": "www-someDomain-net",
+        "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1/customdomains/www-someDomain-net",
+        "type": "Microsoft.Cdn/profiles/endpoints/customdomains",
+        "properties": {
+            "provisioningState": "Succeeded",
+            "resourceState": "Active",
+            "hostName": "www.someDomain.net",
+            "customHttpsProvisioningState": "Enabling",
+            "customHttpsProvisioningSubstate": "PendingDomainControlValidationRequestApproval",
+            "validationData": null
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Create.json]: Cannot be null

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Create.json [hunk 32]
```diff
@@ -0,0 +1,30 @@
...
+      "body": {
+        "name": "www-someDomain-net",
+        "id": "/subscriptions/subid/resourcegroups/RG/providers/Microsoft.Cdn/profiles/profile1/endpoints/endpoint1/customdomains/www-someDomain-net",
+        "type": "Microsoft.Cdn/profiles/endpoints/customdomains",
+        "properties": {
+            "provisioningState": "Succeeded",
+            "resourceState": "Active",
+            "hostName": "www.someDomain.net",
+            "customHttpsProvisioningState": "Enabling",
+            "customHttpsProvisioningSubstate": "PendingDomainControlValidationRequestApproval",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Create.json]: Mismatch `PendingDomainControlValidationREquestApproval` in the spec

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Delete.json [hunk 33]
```diff
@@ -0,0 +1,16 @@
...
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
+      "body": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Delete.json]: Remove empty body

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Delete.json [hunk 34]
```diff
@@ -0,0 +1,16 @@
...
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_Delete.json]: Add responses for 202 and 204.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_DisableCustomHttps.json [hunk 35]
```diff
@@ -0,0 +1,16 @@
...
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
+      "body": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_DisableCustomHttps.json]: Remove empty body

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_DisableCustomHttps.json [hunk 36]
```diff
@@ -0,0 +1,16 @@
...
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_DisableCustomHttps.json]: Add 202 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_EnableCustomHttps.json [hunk 37]
```diff
@@ -0,0 +1,16 @@
...
+  "parameters": {
+    "api-version": "2017-10-12",
+    "subscriptionId": "subid",
+    "resourceGroupName": "RG",
+    "profileName": "profile1",
+    "endpointName": "endpoint1",
+    "customDomainName": "www-someDomain-net"
+  },
+  "responses": {
+    "200": {
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/CustomDomains_EnableCustomHttps.json]: Add 202 response.

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/ValidateProbe.json [hunk 38]
```diff
@@ -0,0 +1,17 @@
+{
+  "parameters": {
+    "api-version": "2017-10-12",
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/ValidateProbe.json]: Add `subscriptionId`

File: specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/ValidateProbe.json [hunk 39]
```diff
@@ -0,0 +1,17 @@
...
+    "validateProbeInput": {
+      "probeURL": "https://www.bing.com/image"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "isValid": true,
+        "errorCode": "None",
+        "message": null
```
Comment (jianghaolu) [inline specification/cdn/resource-manager/Microsoft.Cdn/2017-10-12/examples/ValidateProbe.json]: Avoid using null in example response properties

**Review by jianghaolu — CHANGES_REQUESTED** [review]

**Comment by jianghaolu** [comment]
There are a number of errors in the examples. Please address. Thanks!

**Comment by prakharsharma10** [comment]
Thanks @jianghaolu for the feedback. There is already a work item [here](https://github.com/Azure/azure-rest-api-specs/issues/2116) for these errors which we are not targeting in this PR.

**Pull request merged**